### PR TITLE
Add cart inventory components

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Add updates inventory items to display products related to the cart.

## Justify why they are needed

These base components could be reused in the cart, the cart notification, the checkout, etc.

## Checklist before requesting a review

- [x] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #763** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/763" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/763?utm_source=stack-comment).

File: apps/store/src/components/CartInventory/OfferInventoryItem.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+type Props = {
+  offer: ProductOfferFragment
+  onRemove?: (offer: ProductOfferFragment) => void
+}
+
+export const OfferInventoryItem = ({ offer, onRemove }: Props) => {
+  const currencyFormatter = useCurrencyFormatter(offer.price.currencyCode)
+
+  return (
+    <InventoryItem.Root>
```
Comment (alebedev) [inline apps/store/src/components/CartInventory/OfferInventoryItem.tsx:16]: Great readability here!  Reusable components help a lot

File: apps/store/src/components/CartInventory/OfferInventoryItem.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <InventoryItem.Left>
+        <Pillow size="small" fromColor="blue" toColor="green" />
+      </InventoryItem.Left>
+      <InventoryItem.Main>
+        <InventoryItem.MainLeft>
+          <p>{offer.variant.displayName}</p>
+        </InventoryItem.MainLeft>
+        <InventoryItem.MainRight>
+          {currencyFormatter.format(offer.price.amount)}/mån
```
Comment (alebedev) [inline apps/store/src/components/CartInventory/OfferInventoryItem.tsx:25]: Nit: we may want some utility for price per period formatting in the future

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/components/CartInventory/OfferInventoryItem.tsx (hunk 2 above)
Comment (robinandeer) [reply apps/store/src/components/CartInventory/OfferInventoryItem.tsx:25]: Def! Please add that to the backlog :)

**Pull request merged**